.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,17 @@
+name: e2e test
+on: push
```

Comment:
This is ok but just wanna make sure this doesn't kill your plan. Does github allow free actions?

Reply:
I thought there were limits on the number of actions you could run per account/repo. But I could be wrong 🤷

Reply:
Ok, just wanted to make sure we weren't going to bombard your personal acct 😅